ci-operator/step-registry/aws/deprovision/appsdnsrecord/aws-deprovision-appsdnsrecord-commands.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Can we extend the ref with more functionality just like what we did in flexy template, maybe we can use it to depression all the recorded cf stacks?  So that we do not have to introduce more similar different ref yam files.

Reply:
replace `aws-deprovision-vpc`, `aws-deprovision-bastionhost`, `aws-deprovision-appsdnsrecord` steps with 
```
aws-deprovision-stacks
aws-deprovision-s3bucket
```